test/bugs-reported/bug3.chpl

```diff
@@ -0,0 +1,8 @@
+proc max(a:int, b:int): int{
```

Comment:
I believe this would also be more appropriate as a JIRA issue.

In case you didn't know, you can accomplish behavior with if/then:
```chpl
proc max(a:int, b:int) {
  return if a > b then a else b;
}
```